### PR TITLE
[UI update] Websites to zones

### PR DESCRIPTION
**Pull request by kodster28**
PCX-7610

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -34,12 +34,6 @@ jobs:
         run: pnpm run crawl
         if: success()
 
-      - name: Check - Broken header links
```
Comment (kodster28) [inline .github/workflows/ci.yml:37]: Removing a check that's no longer necessary

File: content/workers/configuration/routing/routes.md [hunk 2]
```diff
@@ -61,7 +61,7 @@ routes = [
 ]
 ```
 
-Add the `zone_name` or `zone_id` option after each route. The `zone_name` and `zone_id` options are interchangeable. If using `zone_id`, find your zone ID by logging in to the [Cloudflare dashboard](https://dash.cloudflare.com) > select your account > select your website > find the **Zone ID** in the lefthand side of **Overview**.
+Add the `zone_name` or `zone_id` option after each route. The `zone_name` and `zone_id` options are interchangeable. If using `zone_id`, find your zone ID by logging in to the [Cloudflare dashboard](https://dash.cloudflare.com) > select your account > select your zone > find the **Zone ID** in the lefthand side of **Overview**.
```
Comment (abstergail) [inline content/workers/configuration/routing/routes.md]: "...in the lefthand side of **Zone Overview**."

File: data/learning-paths/application-security.json [hunk 3]
```diff
@@ -52,7 +52,7 @@
       {
         "type": "module",
         "title": "Before you begin",
-        "description": "Before you can secure your site, make sure you have already <a href='/learning-paths/get-started/'>added that site to Cloudflare</a>.",
+        "description": "Before you can secure your website or application, make sure you have already <a href='/learning-paths/get-started/'>added that site to Cloudflare</a>.",
```
Comment (abstergail) [inline data/learning-paths/application-security.json]: "...added it as a Cloudflare zone."

File: data/learning-paths/optimize-site-speed.json [hunk 4]
```diff
@@ -42,7 +42,7 @@
       {
         "type": "module",
         "title": "Before you begin",
-        "description": "Before you can secure your site, make sure you have already <a href='/learning-paths/get-started/'>added that site to Cloudflare</a>.",
+        "description": "Before you can secure your website or application, make sure you have already <a href='/learning-paths/get-started/'>added that site to Cloudflare</a>.",
```
Comment (abstergail) [inline data/learning-paths/optimize-site-speed.json]: "...added it as a Cloudflare zone."

File: content/fundamentals/get-started/setup/add-site.md [hunk 5]
```diff
@@ -1,10 +1,10 @@
...
-title: Add a site
+title: Add a website
 pcx_content_type: tutorial
 weight: 2
 ---
 
-# Add site to Cloudflare
+# Add a website or application to Cloudflare
 
 When you add a site to Cloudflare, you need to create a new domain within Cloudflare and then perform additional steps to activate that domain.
```
Comment (abstergail) [inline content/fundamentals/get-started/setup/add-site.md]: can we also update this to:
"When you add a website or application to Cloudflare, you need to create a new Cloudflare zone and perform additional steps to activate the zone."

File: data/learning-paths/application-security.json [hunk 6]
```diff
@@ -2,7 +2,7 @@
     "path": "/learning-paths/application-security/",
```
Comment (abstergail) [inline data/learning-paths/application-security.json:2]: Should step 2 of this link directly to fundamentals/get-started/setup/add-site/ instead?

File: content/workers/configuration/routing/routes.md (hunk 2 above)
Comment (kodster28) [reply content/workers/configuration/routing/routes.md]: addressing in another way

**Review by deadlypants1973 — APPROVED** [review]

**Review by haleycode — APPROVED** [review]

**Pull request merged**